File: fm-server/src/main/java/io/zeitwert/fm/server/session/adapter/rest/dto/LoginResponse.java
package io.zeitwert.fm.server.session.adapter.rest.dto;

import io.zeitwert.dddrive.ddd.api.rest.dto.EnumeratedDto;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class LoginResponse {

	private final String tokenType = "Bearer";
	private String token;

	private Integer id;
	private String username;
	private String email;
	private Integer accountId;
	private EnumeratedDto role;

}

File: fm-server/src/main/java/io/zeitwert/fm/server/session/adapter/rest/dto/LoginResponse.kt
package io.zeitwert.fm.server.session.adapter.rest.dto

import io.zeitwert.dddrive.api.jsonapi.dto.EnumeratedDto

data class LoginResponse(
	val sessionId: String? = null,
	val id: Int? = null,
	val username: String? = null,
	val email: String? = null,
	val accountId: Int? = null,
	val role: EnumeratedDto? = null,
)
